### PR TITLE
chore: fix CI

### LINKED ISSUE #1: CI fails
Error: https://github.com/Agoric/dapp-amm/runs/4889294435?check_suite_focus=true

```
./src/utils/config.js
Cannot find file '../generated/defaults' in './src/utils'.
```

We can fix this by checking in `/generated/defaults` like in https://github.com/Agoric/dapp-treasury/commit/99381fd0e46fb8a8415bc3f7faefe55589218572#diff-adbd19f4ef3c2c3f5c6b0598aa6ac1c43bdd85f29ad514e54f60334e7f11d114

### PR DESCRIPTION
**Pull request by samsiegart**
fixes: https://github.com/Agoric/dapp-amm/issues/1

### CODE REVIEW

File: .github/workflows/deploy-treasury.yml [hunk 1]
```diff
@@ -65,35 +65,35 @@ jobs:
 #    outputs:
 #      deployments: ${{ steps.set-deployments.outputs.deployments }}
 
-  deploy:
```
Comment (michaelfig) [inline .github/workflows/deploy-treasury.yml]: I'd prefer renaming `deploy-treasury.yml` to `deploy-amm.yml.DISABLED`.

File: ui/src/components/Liquidity/LiquidityPool/ItemLiquidityPool.jsx [hunk 2]
```diff
@@ -19,7 +18,7 @@ const ItemLiquidityPool = ({
   const { state } = useApplicationContext();
   const { assets } = state;
 
-  const [error, setError] = useContext(ErrorContext);
+  const [_, setError] = useContext(ErrorContext);
```
Comment (michaelfig) [inline ui/src/components/Liquidity/LiquidityPool/ItemLiquidityPool.jsx]: I'd prefer not using just `_` as it pollutes the local namespace.

```suggestion
  const [_error, setError] = useContext(ErrorContext);
```

File: ui/src/components/Swap/DialogSwap/AssetDialog.jsx [hunk 3]
```diff
@@ -86,7 +86,7 @@ const AssetDialog = ({ type, setSelectedAsset }) => {
       <div className="flex flex-col gap-4 p-5 overflow-auto ">
         {Array(4)
           .fill({})
-          .map(item => (
+          .map(__ => (
```
Comment (michaelfig) [inline ui/src/components/Swap/DialogSwap/AssetDialog.jsx]:
```suggestion
          .map(() => (
```

File: ui/src/components/Swap/DialogSwap/PurseDialog/SkeletonPurseDialog.jsx [hunk 4]
```diff
@@ -15,7 +15,7 @@ const SkeletonPurseDialog = () => {
       <div className="flex flex-col gap-4  px-5 pb-5 overflow-auto ">
         {Array(3)
           .fill({})
-          .map(item => (
+          .map(_ => (
```
Comment (michaelfig) [inline ui/src/components/Swap/DialogSwap/PurseDialog/SkeletonPurseDialog.jsx]:
```suggestion
          .map(() => (
```

**Review by michaelfig — APPROVED** [review]
Minor cleanups suggested.  Feel free to merge when you've decided to implement or ignore.

File: .github/workflows/deploy-treasury.yml (hunk 1 above)
Comment (samsiegart) [reply .github/workflows/deploy-treasury.yml]: Done

**Comment by samsiegart** [comment]
Thanks for review!

**Pull request merged**